kernel/smp.c

```diff
@@ -115,10 +115,21 @@ static inline FUNC_NORETURN void smp_init_top(void *arg)
 	struct k_thread dummy_thread;
 	struct cpu_start *csd = arg;
 
+	/* Let the CPU start caller know that this CPU
+	 * has powered up and ready to be initialized.
+	 */
```

Comment:
I wish I could think of a more concise phrasing than "the CPU start caller", but we can polish it slightly with "Inform xxx" instead of "Let xxx know".

Also, "and ready" -> "and is ready"

Reply:
Updated.